rbenchmarker/benchmarker/README.md

```diff
@@ -0,0 +1,45 @@
+# Generating test data
+
+The test data can be generated by building the docker image defined and running it:
+
+```bash
+docker build -t benchmarker_r .
+#Print the helps for the supported arguments
+docker run benchmarker_r -h
+#run the benchmarker leaving most arguments as default
+docker run -v /path/to/data:/data -v /path/to/output:/output benchmarker_r  \
+             --data-path=/data --output-path=/output - --max-percent-of-rows=0.01 --verbose
+```
+
+The image has an entrypoint that expects two major arguments:
+
+- --data-path: The path, within the docker container, to the sources data to test. It also expects the `test.yaml` file that defines the details of test data.
+- --output-path: The path, again within the docker container, where the benchmark output should be written.
+- --max-percent-of-rows:  the percentage of the total number of rows of the data. It is used to set the upper bound of size of region to select. Default is 0.1. To get a quick test run, set it to the smaller size (e.g. 0.001)
```

Comment:
It might be helpful to describe how the test you're running works, otherwise this argument is a bit confusing.

Reply:
done. Let me know if more clarifications are still needed.